RUFAS/biophysical/animal/digestive_system/digestive_system.py

```
@@ -102,14 +118,11 @@ def process_digestion(self, digestive_system_inputs: DigestiveSystemInputs) -> N
             return
 
         else:
-            supported_animal: list[str] = ["Calf", "HeiferI", "HeiferI", "HeiferII", "HeiferIII", "DryCow", "LacCow"]
-            info_map = {
-                "class": DigestiveSystem.__name__,
-                "function": DigestiveSystem.process_digestion.__name__,
-            }
             om.add_error(
-                "Unsupported animal type",
-                f"Supported animal types are {supported_animal}. Got {digestive_system_inputs.animal_type}",
+                "Unexpected execution path in process_digestion evaluating animal type",
+                f"Supported animal types are {supported_animals}. Got {digestive_system_inputs.animal_type}",
                 info_map,
             )
-            raise TypeError("Unsupported animal types")
+            raise RuntimeError(
+                f"Unexpected execution path in process_digestion. Animal type: {digestive_system_inputs.animal_type}"
+            )
```

Added a final else path of a runtime error to ensure something happens and it's not failing silently.